README.md

```diff
@@ -73,3 +73,30 @@ Note: If you are using more then one type of component style files (ie a .less f
  - [`ember-cli-sass`](https://github.com/aexmachina/ember-cli-sass)
  - [`ember-cli-less`](https://github.com/gdub22/ember-cli-less)
  - [`ember-cli-stylus`](https://github.com/drewcovi/ember-cli-stylus)
+
+### Configuration
+
+You can set the following configuration options in your `config/environment.js` file:
+
+```js
+ENV['ember-component-css'] = {
+  option: 'value'
+}
+```
+
+**namespacing(_enabled_)**
+
+Defaults to true. Set this option to `false` to disable the namespacing feature of Ember Component CSS.
+
+```js
+ENV['ember-component-css'] = {
+  namespacing: false
+}
+```
+
+If `false`, the default behavior changes in two ways:
+
+ 1. The autogenerated component id is no longer added to your component's HTML
+ 2. Your pod CSS files are no longer are namespaced using the autogenerated component id.
+
+Essentially, this tells `ember-component-css` to just include all the `styles.scss` files it finds in your `/app` directory into your final css output.
```

Comment:
it only looks for files that end in one of the css extensions that have been defined. So can be any css extension that has been registered.. `css, sass, scss, less, etc`.. and it looks in your pods directory, and if you don't have one defined, then your app. And you have to explicitly import the `pod-styles` file into you css.